htdocs/societe/class/address.class.php

```diff
@@ -31,21 +31,21 @@
  */
 class Address
 {
-	var $db;
-
-	var $id;
-	var $type;
-	var $label;
-	var $socid;
-	var $name;
-	var $address;
-	var $zip;
-	var $town;
-	var $country_id;
-	var $country_code;
-	var $phone;
-	var $fax;
-	var $note;
+	private $db;
```

Comment:
When I extend from dolibarr classes, I use $this->db in my extended classes. So protected.